### PR TITLE
chore: update the integration editor routes to accommodate all the flows 

### PR DESCRIPTION
**Pull request by riccardo-forina**
Following the specs [here](https://docs.google.com/document/d/1Wcg2RQCisBin7gMFfdQ2J7lQDiNc18yw_gtZwvs7NF8/edit?ts=5cb5f65d#heading=h.yese8dam9mdw)

### CODE REVIEW

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/670/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

File: app/ui-react/syndesis/src/modules/integrations/pages/create/configure/main/SaveIntegrationPage.tsx [hunk 1]
```diff
@@ -54,16 +54,17 @@ export class SaveIntegrationPage extends React.Component {
...
-                  defaultValue: '',
-                  displayName: 'Description',
-                  type: 'textarea',
-                },
                 name: {
                   defaultValue: '',
                   displayName: 'Name',
                   required: true,
                 },
+                // tslint:disable-next-line
```
Comment (gashcrumb) [inline app/ui-react/syndesis/src/modules/integrations/pages/create/configure/main/SaveIntegrationPage.tsx]: You can add `order` to these fields to organize them how they should be, so no need to disable tslint.  Set the `name` to have `order: 0` and the description to have `order: 1`

Comment (riccardo-forina) [reply app/ui-react/syndesis/src/modules/integrations/pages/create/configure/main/SaveIntegrationPage.tsx]: Oh good call, thanks! I'll probably do both things, it helps me see things the way they should be.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/683/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/690/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/694/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Pull request merged**